unittest/baseapi_thread_test.cc

```diff
@@ -23,7 +23,9 @@
 #include <functional>
 #include <memory>
 #include <string>
+#ifdef INCLUDE_TENSORFLOW
```

Comment:
Is it used only for thread pool in this test?
If yes, we should rewrite it to c++11 or abseil.

Reply:
https://github.com/abseil/abseil-cpp/blob/master/absl/synchronization/internal/thread_pool.h

Reply:
Yes. I'd prefer c++11 and reduce the use of abseil (which causes lots of compiler warnings).